yqlplus_engine/src/main/java/com/yahoo/yqlplus/engine/internal/plan/types/base/BaseTypeWidget.java

```diff
@@ -121,6 +121,14 @@ public boolean isAssignableFrom(TypeWidget type) {
                 if (type instanceof ListTypeWidget) {
                     return true;
                 } else {
+                    try {
+                        Class clazz1 = Class.forName(getJVMType().getClassName());
+                        Class clazz2 = Class.forName(type.getJVMType().getClassName());
+                        if (clazz1.isAssignableFrom(clazz2)) {
```

Comment:
This test (getting the Class instances and using the built-in isAssignableFrom) would also apply to the later clause for OBJECT.

Reply:
added fix for OBJECT